packages/std/src/errors.rs

```diff
@@ -43,6 +43,14 @@ pub enum StdError {
         source: serde_json_wasm::ser::Error,
         backtrace: snafu::Backtrace,
     },
+    #[snafu(display("Unauthorized"))]
+    Unauthorized { backtrace: snafu::Backtrace },
+    #[snafu(display("Cannot subtract {} from {}", subtrahend, minuend))]
+    Underflow {
+        minuend: u128,
+        subtrahend: u128,
+        backtrace: snafu::Backtrace,
+    },
```

Comment:
- Do we have any strategy regarding the `*Err` suffix?
- Can we convert `minuend`/`subtrahend` to String, allowing us to use the error type in more cases?

Reply:
Sure, I will use String here.